### ISSUE TITLE
LM 21 GTK-4.0 Mint-Y: Widget Factory shows around a half-dozen issues

### ISSUE THREAD

**Issue by SebastJava** [linuxmint/mint-themes#398]
**Many issues in GTK-4.0 Widget Factory on LM 21 with Mint-Y**  
- [ ] Round corners on maximized windows
- [ ] Black square backgrounds on window controls on hover
- [ ] Black square background on window close button
- [ ] Buggy dark mode
- [ ] No margins around buttons in pop-up dialogs
- [ ] Bad animation for showing end or beginning when scrolling
- [ ] Missing lines on treeview
- [ ] "GTK could not find a media module"

**Or maybe this is just my problem caused by my virtual machine?**
* up-to-date system
* `inxi -Fxxxrz > /media/sf_VirtualBox_Shared/VM-inxi-$(date +%Y-%m-%d).txt`: 
[VM-inxi-2022-08-20.txt](https://github.com/linuxmint/mint-themes/files/9388071/VM-inxi-2022-08-20.txt)

**To Reproduce**  
1. `apt install gtk-4-examples`
2. Launch Widget Factory and explore

**Video capture**  
[LM21-GTK4-WF-2022-08-20_18.40.01.webm](https://user-images.githubusercontent.com/31940713/185768455-02668f30-e94c-41a3-b7d1-1005ab965b20.webm)

**Desktop**  
Linux Mint Cinnamon 21

**Comment by SebastJava** [comment]
P.S.: I would appreciate if anyone could validate or invalidate this as soon as possible. Are these real bugs or is it just me and my virtual machine?

**Comment by SebastJava** [comment]
EDIT: multi-issue OP converted to task list.

**Comment by JosephMcc** [comment]
1: fixed
2: can't reproduce
3: sounds the same as 2 and can't reproduce
4: not a theme issue
5: fixed
6: Can look at improving but not a but
7: fixed
8: not a theme issue

**Comment by SebastJava** [comment]
@JosephMcc 
Sorry to insist but I still have 2 out of 8 multi-issues in an undefined or unclear state.

I synced my GitHub source branch to your latest commits just now, and pulled it on my local HDD. Then I have build the package and updated my VM. And then I refreshed cinnamon-settings themes and re-opened WidgetFactory-GTK-4...

So, I can confirm 3 things got fixed, and I understand 2 others are not under your responsibility. And we will leave task number 6 for later. Please check my updated task list here on top about these updates and precisions. ↑

**So, this still leaves 2 similar issues in an unclear state:**
* Black square backgrounds on window minimize-maximize buttons on hover
* Black square background on window close button

**Video capture**
[LM21-GTK4-WF-2022-08-21_13.23.55.webm](https://user-images.githubusercontent.com/31940713/185804307-a200b23b-d03a-4cbf-b9b0-0c5b211953d3.webm)

TO SUM UP
We are talking about LM 21 WidgetFactory-GTK-4.0 with the Mint-Y theme. Do you confirm you are not getting this bug showed here in this latest video? This issue could be caused by my VirtualBox, but that would be the first time. So far, my latest VM install was 100% accurate on all LM 21 beta testing. I can only find two possible explanations: either I was not clear at describing those 2 bugs, or these 2 bugs comes from my VM.

**Comment by SebastJava** [comment]
I made a very short video right here, just for those 2 issues...

**Comment by SebastJava** [comment]
* Black square backgrounds on window minimize-maximize buttons on hover
* Black square background on window close button

![bad-window-controls](https://user-images.githubusercontent.com/31940713/186234749-fa7615f3-1857-4b33-b5d4-970c820129a9.png)

**Comment by SebastJava** [comment]
@JosephMcc **Okay. No bugs.**
I did another test, this time from a live LM 21 USB stick. On real hardware. AFAIK, these bad looking window controls were caused by my virtual machine. Now it looks okay. And the buggy dark mode is gone too. It all works fine, on real hardware.
